packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.styles.ts

```diff
@@ -55,33 +56,32 @@ export const inputWrapperBaseStyles = css`
 
 export const inputWrapperModeStyles: Record<Theme, string> = {
   [Theme.Light]: css`
-    color: ${palette.black};
-    background: ${palette.white};
-    border: 1px solid ${palette.gray.base};
-
-    &:hover,
-    &:active {
-      &:not(:focus) {
-        box-shadow: ${hoverRing.light.gray};
-      }
-    }
+    color: ${color.light.text.primary.default};
+    background: ${color.light.background.primary.default};
```

Comment:
maybe not worth rewriting in this PR, but now that we have the `colors` token, we can consolidate this into a single function:

```ts
export const getInputWrapperModeStyles = (theme: Theme) => css`
    color: ${color[theme].text.primary.default};
    background: ${color[theme].background.primary.default};
`
```

Reply:
I started applying this pattern where I could. Unfortunately, until we get design to align on how to handle dark mode backgrounds using gray.dark4 which is currently the value of `color.dark.background.secondary.default` rather than `color.dark.background.primary.default`, I think it'll be better to retain the `Theme` object style for these cases

We've got this design ticket to figure out how to handle those background tokens: https://jira.mongodb.org/browse/LG-4226